tecan_od_analyzer/__main__.py

```diff
@@ -697,26 +577,148 @@ def main():
                                     colnames[col]: clean_series.values,
                                 }
                             )
-                            gr_plots(
-                                df,
-                                colnames[col],
-                                color_=color_palette,
-                                legend_="exclude",
-                                title_="species",
-                                interpolationplot=interpolationplot,
-                                flag_svg=flag_svg,
-                            )
+
+                            # First time
+                            if start_leg == "":
```

Comment:
Is it possible for this to fail when e.g., (colnames[col])[:3] = ""?

Reply:
Added a check to make sure that there is a bioshaker input at this point but the bioshaker identification is added to the name during processing so it would be a surprise if it were not there